Library/Koala/KoalaTests.swift

```diff
@@ -571,4 +571,26 @@ final class KoalaTests: TestCase {
     XCTAssertEqual(["Cool Live Stream"], client.properties(forKey: "live_stream_name", as: String.self))
     XCTAssertEqual([startDate], client.properties(forKey: "live_stream_start_date", as: Double.self))
   }
+
+  func testTrackSentChatMessage() {
```

Comment:
nice! thanks for writing a dedicated koala tracking test! we don't do this enough